src/app/auth/account/additional-components/account-flows-tab/account-flows-tab.component.ts

```diff
@@ -0,0 +1,156 @@
+import { BaseComponent } from "src/app/common/base.component";
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { Observable, combineLatest, switchMap, timer } from "rxjs";
+import { MaybeNull, MaybeUndefined } from "src/app/common/app.types";
+import {
+    CancelFlowArgs,
+    FilterByInitiatorEnum,
+    FlowsTableData,
+} from "src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.types";
+import { DatasetFlowsService } from "src/app/dataset-view/additional-components/flows-component/services/dataset-flows.service";
+import { NavigationService } from "src/app/services/navigation.service";
+import { mockFlowSummaryDataFragments } from "src/app/api/mock/dataset-flow.mock";
+import { Dataset, FlowStatus, FlowSummaryDataFragment, InitiatorFilterInput } from "src/app/api/kamu.graphql.interface";
+import { AccountService } from "src/app/services/account.service";
+import { AccountTabs } from "../../account.constants";
+import ProjectLinks from "src/app/project-links";
+import { requireValue } from "src/app/common/app.helpers";
+
+@Component({
+    selector: "app-account-flows-tab",
+    templateUrl: "./account-flows-tab.component.html",
+    styleUrls: ["./account-flows-tab.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AccountFlowsTabComponent extends BaseComponent implements OnInit {
+    @Input() accountName: string;
+    public tileWidgetData$: Observable<MaybeUndefined<FlowsTableData>>;
+    public nodes: FlowSummaryDataFragment[] = [mockFlowSummaryDataFragments[0]];
+    public filterByStatus: MaybeNull<FlowStatus> = null;
+    public filterByInitiator = FilterByInitiatorEnum.All;
+    public searchByDataset: MaybeNull<Dataset> = null;
+    public currentPage = 1;
+
+    public flowConnectionData$: Observable<[FlowsTableData, Dataset[], FlowsTableData, MaybeUndefined<boolean>]>;
+    public readonly WIDGET_FLOW_RUNS_PER_PAGE: number = 150;
+    public readonly TABLE_FLOW_RUNS_PER_PAGE: number = 15;
+    public readonly FlowStatus: typeof FlowStatus = FlowStatus;
+    public readonly TIMEOUT_REFRESH_FLOW = 800;
+
+    constructor(
+        private flowsService: DatasetFlowsService,
+        private navigationService: NavigationService,
+        private accountService: AccountService,
+    ) {
+        super();
+    }
+
+    ngOnInit(): void {
+        this.getPageFromUrl();
+        this.getTableDate(this.currentPage);
+    }
+
+    private getTableDate(
```

Comment:
data, not date

Reply:
Method called `get` should return something. This is not `get`, this is `update` or `fetch`.

Reply:
Fixed